lib/rubocop/cop/mixin/frozen_string_literal.rb

```diff
@@ -35,6 +35,10 @@ def frozen_string_literals_enabled?
         leading_comment_lines.any? { |line| MagicComment.parse(line).frozen_string_literal? }
       end
 
+      def frozen_string_literals_disabled?
```

Comment:
Could you please explain, why we're not able to use previous method `frozen_string_literals_enabled?`. In case we can't it even looks strange because we check leading comment for `MagicComment.parse(line).frozen_string_literal?` twice

Reply:
Because the `frozen_string_literal` value can be anything, and is only cast to `true` or `false` if it is literally one of those strings. I probably should change the `disabled?` method to check `frozen_string_literal == false`.